src/MBC3.md

```diff
@@ -58,11 +60,23 @@ Controls what is mapped into memory at A000-BFFF.
 
 ### 6000-7FFF - Latch Clock Data (Write Only)
 
-When writing $00, and then $01 to this register, the current time
-becomes latched into the RTC registers. The latched data will not change
-until it becomes latched again, by repeating the write $00-\>$01
-procedure. This provides a way to read the RTC registers while the
-clock keeps ticking.
+Latching makes a static copy of the current timestamp available in the clock counter registers while the clock keeps running in the background. This makes sure that your reads from the counter registers will be consistent, since any counter overflowing while you read the different parts can have you read an incorrect value (e.g. reading the hour at 11:59 and the minute at 12:00 will give 12:59.)
+
+The exact behavior of this register varies depending on hardware:
+
+MBC3B provides a running clock on power-on and after writing any even value to this register. It is still recommended to latch the clock by writing any odd value. MBC3B can only latch while it provides a running clock, so you must write an even value before you can write an odd value again.
```

Comment:
I don't think the two versions of this chip are mentioned before this point? I'd suggest adding a paragraph explaining this in the intro section.